core/web/frontend/staticfiles/yeti/js/filedrag.js

```diff
@@ -0,0 +1,62 @@
+$(function () {
+
+  $("#dropzone").on('dragenter', function(e) {
+    console.log('asd');
+    $(this).addClass("dragging");
+  });
+
+  $("#dropzone").on('dragleave', function(e) {
+    console.log('asd');
+    $(this).removeClass("dragging");
+  });
+
+  $("#dropzone input").on('change', function(e) {
+    var files = $(this)[0].files;
+    var span = $("#dropzone span");
+    if (files.length > 1) {
+      span.html("<strong>" + files.length + "</strong> files selected");
+    }
+    else if (files.length == 1) {
+      span.html("Selected <strong>" + files[0].name + "</strong>");
+    }
+  });
+
+
+  $("form.dropzone").on('submit', function(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    form = $(this)
+    form.find(".btn.upload").prop('disabled', true)
+    form.find('span').html("<strong>Uploading...</strong>")
+    ajaxFormData = new FormData();
+
+    files = form.find("input[type=file]")[0].files
+
+    for (var i = 0; i < files.length; i++) {
+      console.log(files[i])
+      ajaxFormData.append("files", files[i], files[i].name);
+    }
+
+    ajaxFormData.append("unzip", form.find(".checkbox.zip input").prop('checked'))
+
+    console.log($(this).attr('action'))
+    $.ajax({
+      method: "POST",
+      headers: {"Accept": "application/json"},
+      processData: false,
+      contentType: false,
+      url: $(this).attr('action'),
+      data: ajaxFormData,
+      success: function(data) {
+        $("#file-upload-modal").modal('toggle');
+        notify("Succesfully uploaded " + data.length + " files.", "success");
+        form.find('span').html("<strong>Drag &amp; drop</strong> or <strong>click</strong> to add files")
+        form.find("input").val('');
+        form.find(".btn.upload").prop('disabled', false)
+      }
+    });
+
+  })
+
+})
```

Comment:
> console.log statements should be removed.

Done.

> I found it weird to only have a "Sucessfully uploaded". I was expecting to be redirected to the search results page.

I hear you. I was thinking that the user may want to upload more files or add other observables, they don't have to go back. OTOH, Displaying search results could be useful in the case where the user wants to check if some files are already included in the database.

> Also kind of weird that this is a modal when "Search & Add" is a page, but this is a detail 😄

We could remove the button from the menu and include it in the "Seach & Add" page, next to `Launch` for example, if that makes more sense to you.